### ISSUE TITLE
Replicate the results

### ISSUE THREAD

**Issue by luisfernandes9** [670768312/MCBTNet#3]
Dear authors, 

I tried to replicate your results in the CVC dataset with the model  defined in the following script : https://github.com/670768312/MCBTNet/blob/main/networks/FPN_FCS_bra_unet_system_atten.py

Unfortunattly, it seems to not work. My suspecion is that the softmax activation in the line 358 of the function features_to_predictions makes it chalenging the model convergnence during the training of binary segmentation (where a sigmoid function is used to calculate the dice and expects logits).

I hope I am wrong 😅.

Best regards.
